### ISSUE TITLE
Image Segmentation.

### ISSUE THREAD

**Issue by R1234A** [YudeWang/UNet_segmentation#1]
I have tried your Repo. to do Image Segmentation in Satellite Data. But I am not getting result when testing on my images.

Can you please check the input image and output as well :

Input Image -

Output - 
![1_result](https://user-images.githubusercontent.com/24264853/55475793-81f23b00-5632-11e9-93b6-4f51fdd1cc23.jpg)

Please let me know what I am doing wrong!!
Thanx in advance.

**Comment by YudeWang** [comment]
I guess the wrong result is caused by the domain shift. The training data is quite different between us. Maybe you should train a model by yourself instead of using pretrained parameter.

**Comment by R1234A** [comment]
Thank you for the quick response. In the train folder, each sample is having 4 png files as : 

1) Mask (Black Image named as 1.png)
2) 1_class.png -- Can you tell me how you have generated this?
3) 1_cls_visible.png -- Can you tell me how you have generated this as well??

Please, can you explain It would be helpful!!

**Comment by YudeWang** [comment]
The repo has been put aside for years :)
Let me guess: 
1. Mask is ground truth. The pixel value is 1-5 (maybe 0-4) and that's why it looks black;
2. Model predicted result (value 0-4 or 1-5);
3. Transfer prediction result into color image(0-255).
